app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -279,6 +283,8 @@ def parse_instance(instance)
           :orchestration_stack => @data_index.fetch_path(:orchestration_stacks, @resource_to_stack[uid]),
           :availability_zone   => @data_index.fetch_path(:availability_zones, 'default'),
           :resource_group      => @data_index.fetch_path(:resource_groups, rg_ems_ref),
+          :labels              => labels,
+          :tags                => map_labels('VmAzure', labels),
```

Comment:
the class should be ``` ManageIQ::Providers::Azure::CloudManager::Vm```

Reply:
that is handled there: https://github.com/ManageIQ/manageiq-ui-classic/pull/3697/files#diff-95b50e997ace14a4e385c9a176eea25fR85
and we discussed it with @cben a lot, and agreed on current implementation, please don't make us starting that over again :)

Reply:
@Ladas this is not supposed to be a class name.  it's just a string that matches what UI puts in ContainerLabelTagMappings (if constrained to one type).  you can see a list of currently used values in Alexander's link ^^ in *keys* of MAPPABLE_ENTITIES const.  see for example `"Image"` which is not a class name.

Yes, this arrangement could be improved as a whole (with a migration — out of scope here), but as it is, adding 1 fully qualified model name to that set would only confuse matters IMO.